conf/docker-dev-env/docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.9"
```

Comment:
I know we've used `/conf/docker` for years but I'm open to alternative locations.

Should the docker-compose file be in the root of the repo?

We would need to flag it as dev-only though.